lib/schema/bigint.js

```diff
@@ -255,7 +255,7 @@ SchemaBigInt.prototype.toJSONSchema = function toJSONSchema(options) {
 };
 
 SchemaBigInt.prototype.autoEncryptionType = function autoEncryptionType() {
-  return 'int64';
+  return 'long';
```

Comment:
The actual bson type strings were not the exact bson type strings that the server expects for the schemas modified in this PR.